examples/nested-params/using-cli/train-nested-cli.py

```diff
@@ -41,4 +41,5 @@
         wandb.log({"loss": _fake_loss})
         time.sleep(0.3)
 
+    # This line is recommended, but not required.
```

Comment:
not recommended

if anything you can do a context manager for wandb.init()... but this shouldnt be needed.